src/components/views/dialogs/InviteDialog.js

```diff
@@ -645,6 +669,14 @@ export default class InviteDialog extends React.PureComponent {
                     } catch (e) {
                         console.warn("Non-fatal error trying to make an invite for a user ID");
                         console.warn(e);
+
+                        // Add a result anyways, just without a profile. We stick it at the
+                        // top so it is most obviously presented to the user.
+                        r.results.splice(0, 0, {
```

Comment:
nit: `unshift` might do the same thing more cleanly

Reply:
I have a personal hatred for `unshift` as a function in javascript (for no reason other than because arrays are overloaded in the language to be stacks, queues, and whatever else you can imagine)